webapp/src/cmds.ts

```diff
@@ -135,6 +135,33 @@ function showUploadInstructionsAsync(
     }).then(() => { });
 }
 
+export function showReconnectDeviceInstructionsAsync(
+    confirmAsync: (options: core.PromptOptions) => Promise<number>
+): Promise<void> {
+    const boardName = pxt.appTarget.appTheme.boardName || lf("device");
+    const helpUrl = pxt.appTarget.appTheme.usbDocs;
+    const jsx = webusb.renderDisconnectDeviceDialog();
+    return confirmAsync({
```

Comment:
![image](https://github.com/user-attachments/assets/085505b8-6e4a-4c03-9280-ba7e3cf1d09d)
happy for feedback on wording, just put together quick one to start with

Reply:
It looks good to me!!